### PR TITLE
[WIP] Bump branch to 3.0.0-dev, drop support for Symfony < 3 and PHP < 5.5.9

### PR DESCRIPTION
**Pull request by romainneutron**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,6 @@
 language: php
```
Comment (sstok) [inline .travis.yml]: 😱 stupid Windows line endings 😅

**Comment by romainneutron** [comment]
See #124

**Comment by pulzarraider** [comment]
I'll be happy if you can support Symfony 2.7 and 2.8. They are LTS versions and we have still many projects on these versions.

**Comment by romainneutron** [comment]
After thinking twice, I think you're right. I propose to change the purpose of this PR and drop support for Symfony version < 2.7 and PHP 5.3.

What do you (anybody) think about this?

File: .travis.yml [hunk 2]
```diff
@@ -14,6 +12,7 @@ before_install:
   - composer self-update
 
 install:
-  - composer install
+  - if [[ $TRAVIS_PHP_VERSION != '5.5' ]]; then composer install ; fi
+  - if [[ $TRAVIS_PHP_VERSION == '5.5' ]]; then composer update --prefer-lowest ; fi
```
Comment (stof) [inline .travis.yml]: IMO, the lowest job should be a dedicated job, also running PHP 5.5 with uptodate deps

File: composer.json [hunk 3]
```diff
@@ -15,8 +15,9 @@
         }
     ],
     "require": {
-        "symfony/framework-bundle": "~2.3|~3.0",
-        "symfony/security": "~2.3|~3.0",
+        "php": ">=5.5.9",
+        "symfony/framework-bundle": "~3.0",
+        "symfony/security": "~3.0",
```
Comment (stof) [inline composer.json]: you should depend on actual security subcomponents instead of using the bigger component

**Comment by stephanvierkant** [comment]
This PR is outdated, but I still think we should drop support for older PHP/Symfony versions. Let's discuss in #246.

**Pull request closed without merging**